src/Plug.vala

```diff
@@ -123,6 +125,13 @@ _("Please connect at least one device to begin configuring the network."), "dial
             main_grid.add (paned);
             main_grid.show_all ();
             add (main_grid);
+
+            if (!client.networking_get_enabled ()) {
```

Comment:
Can we get this if clause into something like `update_networking_state ()` and then just call that?